web/src/pages/Player/Player.jsx

```diff
@@ -24,6 +24,7 @@ const Player = () => {
     );
     const sourceIsInactive = getSourceInputType(selectedSource) === "none";
     const [expanded, setExpanded] = useState(false);
+    const [alone, setAlone] = useState(false);
```

Comment:
Feels like the alone state might be redundant since its derived purely from other states. Could you compute the alone value here instead of in VolumeZones? That way we can get rid of the extra state and VolumeZones would no longer need to take in setAlone. 

Since alone depends on getFittestRep, that would need to be computed in Player and the output passed down to VolumeZones instead of VolumeZones computing it.